external/fv3fit/fv3fit/data/synthetic.py

```diff
@@ -0,0 +1,238 @@
+from .base import TFDatasetLoader, register_tfdataset_loader
+import dataclasses
+from typing import Optional, Sequence, List
+import tensorflow as tf
+import numpy as np
+from ..tfdataset import generator_to_tfdataset
+import dacite
+
+
+@register_tfdataset_loader
+@dataclasses.dataclass
+class SyntheticNoise(TFDatasetLoader):
```

Comment:
Curious what's the intended use of the noise dataset?

Reply:
It's handy for single-sample overfitting tests. I wrote it when I was concerned that waves were particularly difficult for CycleGAN to overfit on, but it turned out CycleGAN is just not good at overfitting.